using-timescaledb/data-retention.md

```diff
@@ -20,6 +27,42 @@ the `conditions` hypertable will still have data stretching back 36 hours.
 For more information on the `drop_chunks` function and related
 parameters, please review the [API documentation][drop_chunks].
 
+### Data Retention with Continuous Aggregates
+
+The function `drop_chunks` can be used to control data retention for
+the continuous aggregate as well, but if you have a continuous
+aggregate defined on the table you need to provide a value for the
+`cascade_to_materialization` parameter. Not giving a value when there is a
+continuous aggregate defined on a hypertable will generate an error.
+
```

Comment:
Could you please add the example and commands here?  i.e. create the view dependent on the condition table and illustrate the error using the example above  `` select drop_chunks(...); ``

Reply:
The normal practice is to *not* show how failing examples, but rather show positive examples, which is shown below. Why do you want to show the negative example?

Reply:
Since you called out the error condition, I asked to illustrate that.
1. Nevertheless, we should add the view definition. The example implicitly assumes that we have a view conditions_summary that depends on conditions. 
2. Also using the example to describe the ignore_invalidation_older_than parameter and the chunks that get dropped would help in clarifying the behavior here. I was wondering if a small table or figure would help show the timelines of what gets dropped and why it errors out. 
3. Would be good to add drop_chunks error for ignore_invalidation_older_than and describe the problem. (i.e. a negative example would be helpful in this case).

Reply:
Ok, I did a significant rewrite of the start of the data retention chapter to provide a running example that progressively shows how to do data retention and also show the error you can run into. I think this is significantly clearer, but please have a look.

Reply:
Note that I did not add a figure. I am creating a figure for timescale/docs.timescale.com-content#350 that outline the complete flow of data.